frontend/packages/console-shared/src/components/dropdown/__tests__/ResourceDropdown.spec.tsx

```diff
@@ -32,7 +32,7 @@ describe('ResourceDropdown test suite', () => {
     const spy = jest.fn();
     const component = shallow(componentFactory({ onChange: spy }));
     component.setProps({ resources: [] });
-    expect(spy).toHaveBeenCalledWith('#CREATE_APPLICATION_KEY#', undefined, true);
+    expect(spy).toHaveBeenCalledWith('#CREATE_APPLICATION_KEY#', undefined, undefined);
```

Comment:
Interesting... pretty sure you don't need to provide `undefined` as the last argument to a function, testing or otherwise... `undefined` is implied by the nature of not providing it in JS.

Reply:
`toHaveBeenCalledWith` function expects the exact number of arguments to be passed.

Reply:
Interesting, fair enough... seemed odd at a glance.